azure-client-authentication/src/main/java/com/microsoft/azure/v2/credentials/AzureCliToken.java

```diff
@@ -72,11 +71,7 @@ String accessToken() {
 
     Date expiresOn() {
```

Comment:
Weird. I guess this class won't be in our library when the new standalone auth library is released, at least.

Reply:
Yeah. I left this one as Date on purpose.